pipeline/db/migrations/20210411135924_populate_benchmarks_new_table.up.sql

```diff
@@ -0,0 +1,34 @@
+-- WIP
```

Comment:
This is a WIP implementation of the migration to move the existing `benchmarks` entries into `benchmarks_new` schema.

I'm still testing this and will finish it in a follow-up PR.